### ISSUE TITLE
"l" question

### ISSUE THREAD

**Issue by choman** [huyng/bashmarks#60]
the current function "l" is typically aliased to something else on Ubuntu 

What's the recommend way to alter the alias for bashmarks, with modifying the code?  i.e. l -> lll

Also, is there a way to reprint the "help" function?

**Comment by AnwarShah** [comment]
I second this. I just unaliased the `l` but i am looking for more neater solution.

**Comment by sam3k** [comment]
I am having the same issue. `l` is for `ls -la` on my environment.

**Comment by turnspike** [comment]
yes, "l" is a very common alias for "ls", ideally there would be a way to prefix all bashmark commands with "b", eg "bl" instead of just "l"

**Comment by felipe-b-oliveira** [comment]
> yes, "l" is a very common alias for "ls", ideally there would be a way to prefix all bashmark commands with "b", eg "bl" instead of just "l".
> 
> Here is a fork that implements it: https://github.com/bachya/bashmarks

How do i add the prefix command? "bm"
